docs/content/preview/yugabyte-platform/configure-yugabyte-platform/supported-os-and-arch.md

```diff
@@ -19,11 +19,13 @@ YugabyteDB Anywhere supports deploying YugabyteDB on both x86 and ARM (aarch64)
 
 YugabyteDB Anywhere supports deploying YugabyteDB on the following operating systems:
 
-* AlmaLinux OS 8
-* CentOS 7
-* Oracle Linux 8
-* Ubuntu 18
-* Ubuntu 20
+* CentOS (default)
+* AlmaLinux OS 8 (tested)
```

Comment:
I think we can get rid of the tested and just leave only the default. If we are listing them on the official supported OS page I would hope they are tested (and all of these have been).

Reply:
Changed